Controller/ApiController.php

```diff
@@ -343,7 +343,7 @@ public function encryptConfigAction(Request $request)
     public function saveConfigAction(Request $request)
     {
         $this->logger->warn("Using deprecated saveConfig call.");
-        /** @var StorageApiService $storage */
+        /** @var OriginalStorageApiService $storage */
```

Comment:
ve skutecnosti je to ale typu Keboola\DockerBundle\Service\StorageApiService

Reply:
jo, asi je, ale dědičnost

Reply:
opraveno